File: Java/Java Fundamentals/Basic Sytax/Exercise Tasks/Task6_StrongNumber.java
import java.util.Scanner;
// 100/100 - може да я пререшиш заради факториела
public class Task6_StrongNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

String inputNum = scanner.nextLine();
int sum = 0;

        for (int i = 0; i < inputNum.length(); i++) {
            int digit = inputNum.charAt(i) - 48;
            int factoriel = 1;
            for (int j = 1; j <= digit ; j++) {

                factoriel *= j;
            }
            sum += factoriel;
        }
            int num = Integer.parseInt(inputNum);
            if (sum == num) {
                System.out.println("yes");
            } else {
                System.out.println("no");
            }



    }
}

File: C#/C# Fundamentals/Basic Syntax, Conditional Statements & Loops/Exercise Tasks/Task6_StrongNumber.cs
using System;
//100/100
namespace Task6_StrongNumber
{
    class Task6_StrongNumber
    {
        static void Main(string[] args)
        {
            string inputNumber = Console.ReadLine();
            int sum = 0;

            for (int i = 0; i < inputNumber.Length; i++)
            {
                int digit = inputNumber[i] - 48;
                int factoriel = 1;
                for (int j = 1; j <= digit; j++)
                {

                    factoriel *= j;
                }
                sum += factoriel;
            }
            int num = int.Parse(inputNumber);
            if (sum == num)
            {
                Console.WriteLine("yes");
            }
            else
            {
                Console.WriteLine("no");
            }

        }
    }
}
